modules/msk/genomenexus/vcf2maf/main.nf

```diff
@@ -0,0 +1,45 @@
+process GENOMENEXUS_VCF2MAF {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'ghcr.io/msk-access/genomenexus:latest':
+        'ghcr.io/msk-access/genomenexus:latest' }"
+
+    input:
+    tuple val(meta), path(vcf)
+
+    output:
+    tuple val(meta), path("vcf2maf_output/${meta.id}.maf"), emit: maf
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    
+
+    """
+    python3 /annotation-tools/vcf2maf.py -i ${vcf} ${args}
```

Comment:
what is the sleep 3 for? Should it be part of args?

Reply:
@buehlere I was using the sleep 3 for testing and forgot to remove, it's been deleted for the next PR commit